install.sh

```diff
@@ -166,7 +261,7 @@ sudo mkdir -p /etc/ros
 
 if [ -e /etc/profile.d/clearpath-ros-environment.sh ]; then
   echo -e "\e[33mWarn: CPR ROS environment exist, skipping\e[0m"
-else
+else1
```

Comment:
Is there is typo?

Reply:
Yes. I apparently fat-fingered my keyboard between testing & saving. I'm fixing that along with another bug I just found.